### PR TITLE
Scaffolds the rails app

### PR DESCRIPTION
**Pull request by fottander**
PT Story: https://www.pivotaltracker.com/story/show/143862501

- Basic rails scaffolding.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
+  gem 'web-console', '>= 3.3.0'
+  gem 'listen', '~> 3.0.5'
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (tochman) [inline Gemfile]: I would clean the `Gemfile` from comment and unused gems.

File: README.md [hunk 2]
```diff
@@ -1 +1,24 @@
...
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
```
Comment (tochman) [inline README.md]: I would add basic project info to the Readme file

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
+  <<: *default
+  database: building_blocks_back_end_production
+  username: building_blocks_back_end
+  password: <%= ENV['BUILDING_BLOCKS_BACK_END_DATABASE_PASSWORD'] %>
```
Comment (tochman) [inline config/database.yml:18]: I would clean this file from comments. Some of them are important so moving them to the `Setup` section of the README would be a good idea.

**Review by diraulo — APPROVED** [review]
:shipit:

**Pull request merged**